test/reject/reject.js

```diff
@@ -0,0 +1,2 @@
+const reject = (pred, array) => array.filter((...args) => !pred(...args))
```

Comment:
Missing semicolon. (Will probably be regenerated if you run the script after updating the `.md` file)